setup.py

```diff
@@ -23,6 +23,7 @@
 max_python_version = "3.12"  # exclusive
 min_numpy_build_version = "1.11"
 min_numpy_run_version = "1.21"
+max_numpy_run_version = "1.25"
```

Comment:
```suggestion
max_numpy_run_version = "1.26"
```

Since we're checking Numpy to be strictly less that max version

Reply:
done in fb03134